python/src/llvm.cc

```diff
@@ -302,6 +303,21 @@ void init_triton_llvm(py::module &&m) {
         PassBuilder pb(nullptr /*targetMachine*/, tuningOptions, std::nullopt,
                        instrCbPtr);
 
+        std::string pluginFile =
+            mlir::triton::tools::getStrEnv("AMDGCN_INSTRUMENTATION_LIB");
```

Comment:
Maybe something like this:
```suggestion
            mlir::triton::tools::getStrEnv("TRITON_INSTRUMENTATION_LIB");
```